src/main/js/all.js

```diff
@@ -1,6 +1,8 @@
 import { WebsocketProvider } from './node_modules/y-websocket/dist/y-websocket.cjs'
 import * as Y from './node_modules/yjs/dist/yjs.cjs'
 
+globalThis.Buffer = require('buffer').Buffer;
```

Comment:
This confirms that we can use the `buffer` npm module.
The proper `Buffer` implementation would be to create a separate js project, pack it, and load it to a polyglot context.